runtime/sync.c

```diff
@@ -0,0 +1,235 @@
+/**************************************************************************/
+/*                                                                        */
+/*                                 OCaml                                  */
+/*                                                                        */
+/*          Xavier Leroy and Damien Doligez, INRIA Rocquencourt           */
```

Comment:
Appears to be a new file. Ensure that the copyright header correctly captures the contributors.

Reply:
I'm not so sure, I essentially moved the code that previously was in systhreads (st_stubs.c). My changes were really to do with naming.

Reply:
Thanks for the clarification.